templates/services/merge/execution/geth.tmpl

```diff
@@ -27,6 +27,7 @@
       - --syncmode=${EC_SYNC_MODE}
       - --http
       - --http.addr=0.0.0.0
+      {{if .Prom}}- --metrics.addr=0.0.0.0{{end}}
       - --http.vhosts=*
```

Comment:
```suggestion
      - --syncmode=${EC_SYNC_MODE}{{if .Prom}}
      - --metrics.addr=0.0.0.0{{end}}
      - --http
      - --http.addr=0.0.0.0
      - --http.vhosts=*
```

Reply:
Rationale:
- Put related flags together (http ones)
- the template if statement should be like this. The way you are doing it leaves unwanted blank spaces/lines